diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index 6bdddfda419e6b533f46f45a90aa8f294c1c7f5c..6d2f419f7c0a55176aa1aa18d93abe49271f09bc 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -2189,6 +2189,7 @@
       // In-development features for the File System Access API.
       name: "FileSystemAccessAPIExperimental",
       status: "experimental",
+      base_feature: "none",
     },
     {
       // The FileSystemHandle.getCloudIdentifiers() method (see
@@ -2203,6 +2204,7 @@
       // Non-OPFS File System Access API.
       name: "FileSystemAccessLocal",
       status: {"default": "stable"},
+      base_feature: "none",
     },
     {
       name: "FileSystemAccessLockingScheme",
@@ -3122,6 +3124,7 @@
     {
       name: "MiddleClickAutoscroll",
       status: "test",
+      base_feature: "none",
     },
     // Killswitch for crbug.com/40062462 fix.
     {
